specification/security/resource-manager/readme.md

```diff
@@ -86,14 +86,24 @@ tag: package-composite-v3
 The following packages may be composed from multiple api-versions.
 
 
+### Tag: package-preview-2022-11
+
+These settings apply only when `--tag=package-preview-2022-11` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2022-11'
+input-file:
+  - Microsoft.Security/preview/2022-11-20-preview/apiCollections.json
+  - Microsoft.Security/preview/2015-06-01-preview/operations.json
```

Comment:
Hmm.. not sure it's required to specify the operations.json - its implicit for every RP to expose it